RockLib.HealthChecks.AspNetCore/Checks/MetricsHealthCheck.cs

```diff
@@ -0,0 +1,143 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using RockLib.HealthChecks.AspNetCore.Collector;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace RockLib.HealthChecks.AspNetCore.Checks;
+
+/// <summary>
+/// Reports HealthCheckMetrics as health check outcomes.
+/// </summary>
+public class MetricsHealthCheck : IHealthCheck
```

Comment:
This should be something more specific like HttpMetricsHealthCheck or similar.

Reply:
Works for me.

Reply:
Renamed [HttpStatsHealthCheck](https://github.com/RockLib/RockLib.HealthChecks/pull/58/commits/ec5460bc3bf1f7856fffe4517d227e2f648229c0)